capten/agent/pkg/capten-store/argocd_projects.go

```diff
@@ -108,13 +108,15 @@ func (a *Store) updateArgoCDProjects() ([]*model.ArgoCDProject, error) {
 	for _, gitProject := range gitProjects {
 		project := &model.ArgoCDProject{Id: gitProject.Id, GitProjectId: gitProject.Id,
 			GitProjectUrl: gitProject.ProjectUrl}
-		if _, ok := argoCDProjects[gitProject.Id]; !ok {
+		if ap, ok := argoCDProjects[gitProject.Id]; !ok {
 			project.Status = string(model.ArgoCDProjectAvailable)
+			project.LastUpdateTime = time.Now().Format(time.RFC3339)
 			if err := a.UpsertArgoCDProject(project); err != nil {
 				return nil, err
 			}
 		} else {
 			project.Status = argoCDProjects[gitProject.Id].Status
+			project.LastUpdateTime = ap.LastUpdateTime
 		}
```

Comment:
add time is store method. check other table handling

Reply:
While inserting the record this time stamp is being added.

Same is being fetched and and returned with this api.